force-app/main/default/lwc/miscLogger/miscLogger.html

```diff
@@ -0,0 +1,26 @@
+<template>
+    <lightning-card title="MiscLogger" icon-name="custom:custom63">
+        <div class="slds-var-m-around_medium">
+            <p class="slds-var-m-bottom_small">
+                <lightning-button
+                    label="Log a message to the console"
+                    onclick={logMessageConsole}
+                ></lightning-button>
+            </p>
+        </div>
+        <div class="slds-var-m-around_medium event-monitoring">
+            <p class="slds-var-m-bottom_small">
+                <lightning-button
+                    label="Log a message to the console and Event Monitoring"
+                    onclick={logMessageEventMonitoring}
+                ></lightning-button>
+            </p>
+        </div>
+
+        <c-view-source source="lwc/misclogger" slot="footer">
+            Log custom messages to Event Monitoring, adding observability to
```

Comment:
Maybe, we can also add a sentence to tell that they can view the output in the console by launching the developer tools.